lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,6 +825,14 @@ def _download_direct_download_link(self, file_url, copy_file_path):
             with closing(online_file):
                 with open(copy_file_path, 'wb') as output:
                     shutil.copyfileobj(online_file, output)
+            # check first 5 lines of file content
+            with open(copy_file_path) as copied_file:
+                for line in islice(copied_file, 5):
+                    if line.lower().find('html') != -1:
+                        raise ValueError("Undownable File.\n" + 
+                            "Please make sure file is publicly accessible\n" +
+                            "File URL: %s" % file_url)
```

Comment:
I don't understand this part at all - it looks like if the first 5 lines of the file contains `html` in any case the file is marked as invalid. That seems completely arbitrary - what if the reads file has `html` in the header somewhere? Also, once this gets moved to DFU, DFU could be downloading any file at all, in which case this check seems nonsensical.